packages/uiweb/src/lib/components/space/SpaceBanner/SpaceBanner.tsx

```diff
@@ -1,15 +1,320 @@
+            <Icon
+              src={
+                getStatus(spaceBannerData?.scheduleAt) === 'Live'
+                  ? live
+                  : getStatus(spaceBannerData?.scheduleAt) === 'Scheduled'
+                  ? scheduled
+                  : '' // Ended
+              }
+              alt="status"
+            />
+            <TimeText>
+              {getStatus(spaceBannerData?.scheduleAt) === 'Live'
+                ? 'Live'
+                : getStatus(spaceBannerData?.scheduleAt) === 'Scheduled'
+                ? `${getDateAndTime(spaceBannerData?.scheduleAt)}`
+                : 'Ended'}
+            </TimeText>
+          </Time>
+          <Participants>
+            <ParticipantsIconContainer orientation={orientation}>
+              {orientation === 'maximized'
+                ? spaceBannerData &&
+                  (spaceBannerData.pendingMembers as []).map(
+                    (person, index) =>
+                      index < 3 && (
+                        <ParticipantsIcon
+                          src={(person as any)?.image}
+                          alt="avatar"
+                          className={`index${index}`}
+                        />
+                      )
+                  )
+                : spaceBannerData &&
+                  (spaceBannerData?.pendingMembers as []).map(
+                    (person, index) =>
+                      index < 2 && (
+                        <ParticipantsIcon
+                          src={(person as any)?.image}
+                          alt="avatar"
+                          className={`index${index}`}
+                        />
+                      )
+                  )}
+            </ParticipantsIconContainer>
+            <ParticipantsText>
+              {orientation === 'maximized'
+                ? spaceBannerData &&
+                  `+${
+                    ((spaceBannerData?.pendingMembers as []).length as number) -
+                    3
+                  }`
+                : spaceBannerData &&
+                  `+${
+                    ((spaceBannerData?.pendingMembers as []).length as number) -
+                    2
+                  }`}
+            </ParticipantsText>
+          </Participants>
+        </Status>
+      </Container>
+    </div>
+  );
 };
+
+// Styling
+const Container = styled.div<{ status?: string; orientation?: string }>`
+  display: flex;
+  flex-direction: ${(props) =>
+    props.orientation === 'maximized' ? 'column' : 'row'};
+  justify-content: ${(props) =>
+    props.orientation === 'maximized' ? 'space-between' : 'space-between'};
+  align-items: ${(props) =>
+    props.orientation === 'maximized' ? 'flex-start' : 'center'};
+  padding: ${(props) =>
+    props.orientation === 'maximized' ? '16px' : '0 11px'};
+  gap: ${(props) => (props.orientation === 'maximized' ? '16px' : '8px')};
+  width: ${(props) =>
+    props.orientation === 'maximized' ? '37vw' : 'fit-content'};
```

Comment:
is this specifying hard-coded width necessary?

Reply:
yes, in order to keep the width of each banner in the feed component, i believe we need to keep it like that.
auto, fit-content, max-content will depend on the title size and will not be uniform imo.